project-base/storefront/components/Blocks/Adverts/Adverts.tsx

```
@@ -33,7 +33,13 @@
                     return <AdvertImage key={advert.uuid} advert={advert} />;
                 }
 
-                return <div key={advert.uuid} dangerouslySetInnerHTML={{ __html: advert.code }} />;
+                return (
+                    <div
+                        key={advert.uuid}
+                        className="focus-visible:ring-actionSecondaryBorder rounded-md focus-visible:ring-2 focus-visible:ring-offset-2"
+                        dangerouslySetInnerHTML={{ __html: advert.code }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)